tests/integration/devfile/cmd_devfile_watch_test.go

```diff
@@ -63,9 +63,10 @@ var _ = Describe("odo devfile watch command tests", func() {
 			// Devfile push requires experimental mode to be set
 			helper.CmdShouldPass("odo", "preference", "set", "Experimental", "true")
 			cmpName := helper.RandString(6)
-			helper.CmdShouldPass("odo", "create", "nodejs", "--project", namespace, "--context", context, cmpName)
+			projectDir := helper.CreateNewContext()
```

Comment:
I think we don't need this. It's already created in the before each.

Reply:
@mik-dass I changed it because we were using the context flag, but staying in the same directory, so it wasn't really testing anything. I changed it so that we'd run those commands outside of the context, making sure `--context` works.

Reply:
I would suggest removing https://github.com/openshift/odo/pull/3090/files#diff-fab0bc73a25f467aa206fd31b58a4c19R34 and changing into the context directory in each test spec or changing to the working directory `currentWorkingDirectory` here.

Reply:
@mik-dass Fair point. I've updated the test so that it changes into `currentWorkingDirectory` before running the `odo watch` command